module_a/src/main/java/org/sonarqube/A.java

```diff
@@ -3,10 +3,12 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](http://localhost:9000/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](http://localhost:9000/project/issues?id=org.sonarqube%3Apr-decoration&branch=pr_33&issues=AWB9UVKNR10kk73Q_CCz&open=AWB9UVKNR10kk73Q_CCz)

[ISSUE_KEY]: AWB9UVKNR10kk73Q_CCz

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED